exe-common/README.md

```diff
@@ -0,0 +1,56 @@
+# Cardano helpers and tooling
+
+[![Build Status](https://travis-ci.org/input-output-hk/rust-cardano.svg?branch=master)](https://travis-ci.org/input-output-hk/rust-cardano)
+![MIT or APACHE-2 licensed](https://img.shields.io/badge/licensed-MIT%20or%20APACHE--2-blue.svg)
+![Cardano Mainnet](https://img.shields.io/badge/Cardano%20Ada-mainnet-brightgreen.svg)
+![Cardano Staging](https://img.shields.io/badge/Cardano%20Ada-staging-brightgreen.svg)
+![Cardano Testnet](https://img.shields.io/badge/Cardano%20Ada-testnet-orange.svg)
+
+Extra toolings and helpers for the [Cardano](https://www.cardano.org) network protocol:
```

Comment:
Extra tools and helpers for the [Cardano](https://www.cardano.org) network protocol: